.sauce/config2Dimension.yml

```diff
@@ -3,7 +3,7 @@ kind: testcafe
 sauce:
   region: us-west-1
   concurrency: 1 # Controls how many suites are executed at the same time.
-  retries: 0
+  retries: 3
```

Comment:
retries / smart retries dont work as far as i tried.

Reply:
While testing the PR for 8.4 I had the case that one run failed and the retry succeeded. But did not enforce it after that.